.golangci.yml

```diff
@@ -7,6 +7,7 @@ run:
   timeout: 10m
   skip-files:
     - ".*\\.pb\\.go"
+    - ".*\\_test.go"
```

Comment:
Why do you need to ignore the test files from the linter? I see that there are only linter issues with `fuzz/ie_fuzz_test.go`. There are no issues with the other test files.

Reply:
The related issue (G404) has been resolved, So I remove the changed line.